### ISSUE TITLE
Long load times

### ISSUE THREAD

**Issue by Shelecktra** [Kopernicus/Kopernicus#399]
Hey I'm a first timer to using Kopernicus, I thought I had done everything right but something isn't working...

Here is the mod I am trying to use.
https://www.curseforge.com/kerbal/ksp-mods/real-solar-system-remastered

I've rolled back KSP to 1.4.5 and I get no load errors when the game is doing its initial load to the main menu.

That's where the problem starts though... The main menu either won't load or I wasn't willing to sit waiting any longer for it to load... Literally sat waiting for almost an hour yesterday with no luck, just a black screen and the loading bar at the bottom right constantly rotating but nothing happening.

Any help would be appreciated, really want to try using some of these planet mods!

**Comment by Sigma88** [comment]
1.4.5 was a long time ago, so I don't really remember what the state of the mod was back then

however, if you could upload the files listed [here](https://forum.kerbalspaceprogram.com/index.php?/topic/140580-SUPPORT/&page=27#comment-2793409) I will take a look and see if there is something specific that might be causing your problem.

please read carefully that post, and make sure you have only the minimum amount of mods when running the game to generate the log files required

**Comment by Shelecktra** [comment]
Thanks for the reply, there's a problem with that link however

**Comment by Sigma88** [comment]
sorry, they probably changed something in the forums since the last time I copied the comment link. [here](https://forum.kerbalspaceprogram.com/index.php?/topic/181547-SUPPORT/&do=findComment&comment=2793409)'s the new link

**Comment by Shelecktra** [comment]
Actually, reading through that has given me some ideas before I spend ages compiling all that for you.

I'm going to reinstall the game fresh, only install kopernicus and the mod I listed above and see if it works. I'm a rather heavy mod user and I can tell from that post that mods seem to cause conflicts, so I'll do that and see if it works and add mods one by one till I find the issue.

If not I'll be back here with the list xD

Thanks!

**Comment by Shelecktra** [comment]
So I appear to still be having the same issue. Here's the info that you requested then!

https://drive.google.com/open?id=1uBtm9H77cXjtF5s2IShEzK_J67L70k9m

There were various cfg files I couldn't find I think but let me know if its important and ill try and find them

**Comment by Galileo88** [comment]
Your issue seems to stem from your VertexHeightMap node. 
`[LOG 15:42:49]: Exception Was Recorded: Object reference not set to an instance of an object
  at PQSMod_VertexHeightMap.OnVertexBuildHeight (.VertexBuildData data) [0x00000] in <filename unknown>:0 `

Check that the path to the texture is correct and that the height map itself is present in the mod. Could also be that it's name is incorrectly spelled in the cfg.

**Comment by Shelecktra** [comment]
Oh, I never realised there was a newer version of it to be honest. I'll try that one and see if I have any luck.

Thanks!

**Comment by Galileo88** [comment]
Make sure you are using KSP 1.6.1. and Kopernicus for 1.6.1, otherwise you will have issues loading the game.

**Comment by Shelecktra** [comment]
Yeah, I found that out the first time I tried kopernicus. Pretty sure im using the right one

**Comment by Sigma88** [comment]
are you still experiencing this issue?

**Comment by Shelecktra** [comment]
Sorry I've been away for a bit!

I actually decided to take the hard line with my pc and just wiped it clean and started again. Had so much stuff I didn't use or need any more anyway. 

Funny thing was when I reinstalled ksp it works fine now. Bit long on the switching to map view and using the tracking station but otherwise it actually works now. Don't know if this was the result of a fresh install or what but yeah, all good now thanks!
